test/plugins/sneak.test.ts

```diff
@@ -0,0 +1,77 @@
+import { getAndUpdateModeHandler } from '../../extension';
+import { Configuration } from '../../src/configuration/configuration';
+import { ModeHandler } from '../../src/mode/modeHandler';
+import { getTestingFunctions } from '../testSimplifier';
+import { cleanUpWorkspace, setupWorkspace } from './../testUtils';
+
+suite('sneak plugin', () => {
+  let modeHandler: ModeHandler;
+  let { newTest, newTestOnly } = getTestingFunctions();
+
+  setup(async () => {
+    await setupWorkspace();
+    modeHandler = await getAndUpdateModeHandler();
+    Configuration.sneak = true;
```

Comment:
FYI, i've made some changes so we can more easily change configurations in the UTs -- https://github.com/VSCodeVim/Vim/pull/2335. PR hasn't yet been merged yet.

Reply:
please resolve the merge conflicts.